packages/conf-llvm/conf-llvm.15/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "Kate <kit.ty.kate@disroot.org>"
+authors: "The LLVM team"
+homepage: "http://llvm.org"
+bug-reports: "https://llvm.org/bugs/"
+license: "MIT"
+build: [
+  ["bash" "configure.sh" version]
+]
+depends: [
+  "conf-bash" {build}
+]
+depexts: [
+  ["llvm@15"] {os-distribution = "homebrew" & os = "macos"}
+  ["llvm-15"] {os-distribution = "macports" & os = "macos"}
+  ["llvm-15-dev"] {os-family = "debian"}
```

Comment:
That looks like a debian bug (it should be a dependency), but oh well
```suggestion
  ["llvm-15-dev" "zlib1g-dev"] {os-family = "debian"}
```

Reply:
cc @sylvestre as you seem to be the LLVM package maintainer in Debian
```
# /usr/bin/ld: cannot find -lz: No such file or directory
```
and links fine if `zlib1g-dev` is installed.
The `-lz` comes from `llvm-config --system-libs --link-static --libs Core`

Reply:
you suggest that I should add this package in the build dep? right?

Reply:
yes, both `zlib1g-dev` and `libzstd-dev`